docs/components/input/README.md

```diff
@@ -436,6 +436,29 @@ Use the `type="number"` attribute only for input fields that reference a numeric
 
 </cdr-doc-example-code-pair>
 
+### Date input
+
+<cdr-banner type="info"><template #icon-left><icon-information-fill inherit-color /></template><span style="font-weight: 500">Vue 3 Only</span></cdr-banner>
+
+Use `type="date"` to create input fields that let a user enter a date, either with a validated text field and/or a simple calendar UI. The presentation is dependent on the browser and largely immutable so this is best used for applications where consistent presentation is less important than the functional benefits.
```

Comment:
Change paragraph to:

Use `type="date"` to create input fields which allow users to enter a date, either with a validated text field and/or a simple calendar UI. Note: the presentation is dependent on the browser and largely unchangeable. Use this input field for applications where consistent presentation is less important than the functional benefits.